go-controller/.golangci.env

```diff
@@ -0,0 +1 @@
+GOLANGCI_LINT_VERSION="v1.52.2"
```

Comment:
ah, I was about to ask I can't see the `GOLANGCI_LINT_VERSION`, where is this defined :) , seems like in the recent push this got added.

Reply:
I can try that at this very same PR

Reply:
I wish I had time to work on this, but I don't right now. I could probably commit to it in a couple sprints, but that's not gonna help I don't think.